### DISCUSSION TITLE
[help] async example

### DISCUSSION THREAD

**Discussion by galr52** [Lodin/react-vtree#74] [Q&A]
so i took the async example from the story but i cant run it 
i have took the example and paste it on web editor

https://react-5zzsnk.stackblitz.io/

any help with whats missing?
any other examples of the async tree?

thanks a lot!

**Comment by Lodin** [comment]
Hey @galr52, sorry for making you wait for so long time. The problem is that you are using the `2.x.x` version that doesn't support that feature. You need to use `3.0.0.-beta.1` for that. 

https://stackblitz.com/edit/react-qvzum3
